### PR TITLE
Adding GeoCod free geocoding API

### PR DESCRIPTION
**Pull request by faaaaaz**
## Requirements

 * [x] This is Software as a Service not self hosted
 * [x] It has a free tier not just a free trial
 * [x] The submission mentions what is free
 * [x] The submission is not already present in the list
 * [x] The service has contact details of those running it and a privacy policy

### CODE REVIEW

**Comment by ripienaar** [comment]
All the ticks needs to be ticked, your service needs the last items.

**Comment by faaaaaz** [comment]
> All the ticks needs to be ticked, your service needs the last items.

Done! Thanks!

**Comment by ripienaar** [comment]
When I say you need to tick all the ticks I dont just mean tick them without actually doing or having the thing the ticks says you need to have.

Please link to your contact details (and not jut info@foo.xyz) and the privacy policy?

Alos fix mardown syntax please

**Comment by faaaaaz** [comment]
> When I say you need to tick all the ticks I dont just mean tick them without actually doing or having the thing the ticks says you need to have.
> 
> Please link to your contact details (and not jut [info@foo.xyz](mailto:info@foo.xyz)) and the privacy policy?
> 
> Alos fix mardown syntax please

do you mean my contact details? or web contact details?

Privacy policy is at home page (bottom).

**Comment by ripienaar** [comment]
Missed the privacy policy there earlier, pls fix the markdown markup

File: README.md [hunk 1]
```diff
@@ -241,7 +241,7 @@ This list is the result of Pull Requests, reviews, ideas and work done by 1100+
   * [FraudLabs Pro](https://www.fraudlabspro.com) — Screen an order transaction for credit card payment fraud. This REST API will detect all possible fraud traits based on the input parameters of an order. Free Micro plan has 500 transactions per month.
   * [FreeGeoIP.app](https://freegeoip.app/) - Completely free Geo IP information (JSON, CSV, XML). No registration required, 15000 queries per hour rate limit.
   * [Geekflare API](https://geekflare.com/api) - Geekflare API let you take screenshots, audit websites, TLS scan, DNS lookup, test TTFB, and a lot more. The free plan offers 3000 API requests.
-  * [GeoCod] https://geocod.xyz) — Free geocoding API: Convert postal addresses into geographic coordinates or convert geographic coordinates into postal addresses (reverse geocoding). 
+  * [GeoCod] (https://geocod.xyz) — Free geocoding API: Convert postal addresses into geographic coordinates or convert geographic coordinates into postal addresses (reverse geocoding).
```
Comment (ripienaar) [inline README.md:244]: no space between `[x](https://x.com)`

**Pull request merged**